src/nv_ingest/stages/pdf_extractor_stage.py

```diff
@@ -18,8 +19,10 @@
 from nv_ingest.schemas.pdf_extractor_schema import PDFExtractorSchema
 from nv_ingest.stages.multiprocessing_stage import MultiProcessingBaseStage
 from nv_ingest.util.exception_handlers.pdf import create_exception_tag
+from nv_ingest.util.logging.configuration import configure_logging
 
 logger = logging.getLogger(f"morpheus.{__name__}")
+configure_logging(logger, os.getenv("INGEST_LOG_LEVEL", "INFO"))
```

Comment:
Should not be required, this will be propagated via: https://github.com/NVIDIA/nv-ingest/pull/14/files#diff-a5b564197d741ea776ee29ab7192991961c613741edc265c9c623c58c0a623cdL657